test/graphql/queries/system_query_test.rb

```diff
@@ -935,27 +957,35 @@ class SystemQueryTest < ActiveSupport::TestCase
 
       asc_result = Schema.execute(
         query,
-        variables: { policyId: @profile1.id, sortBy: ['rulesFailed'] },
+        variables: {
+          policyId: @profile1.id,
+          sortBy: ['rulesFailed'],
+          filter: "policy_id = #{@profile1.id}"
```

Comment:
I updated the filter from `with_results_for_policy_id` to `policy_id` so that we could test the nil rules_failed and it wasn't being returned with `with_results_for_policy_id` (since it doesn't have any results)